tests/tests.rs

```diff
@@ -65,21 +65,15 @@ fn test_sleep() {
 #[test]
 #[should_panic]
 fn test_panic() {
-    let f = || {
-        fail_point!("panic");
-    };
     fail::cfg("tests::panic", "panic(msg)").unwrap();
-    f()
+    fail_point!("panic");
```

Comment:
I think this is confusing. `fail_point` is used to define a fail point, but the usage here is just to evaluate the fail point.